server/fleet/activities.go

```diff
@@ -807,6 +810,50 @@ func (a ActivityTypeEditedMacosProfile) Documentation() (activity, details, deta
 }`
 }
 
+type ActivityTypeChangedMacosSetupAssistant struct {
```

Comment:
Can this be merged even if the frontend is not ready to handle those new activity types? I don't think the frontend will be done for this release. /cc @lukeheath @georgekarrv

Reply:
context: the activity will show up in the UI with the default template, which sometimes is very ugly and sometimes is ok.

Reply:
@roperzh busted them out - thanks! https://github.com/fleetdm/fleet/pull/11323